docs/softwares.md

```
@@ -173,4 +173,27 @@ Conversion of the filtered cnvpytor vcf to AED file format. The AED file can be
 
 #RESOURCESSCHEMA__vcf_to_aed_filtered#
 
+---
+
+## create_somalier_mqc_tsv
+Create MultiQC custom content TSV files from Somalier output. This script processes somalier relatedness and sex check data to create custom tables similar to Peddy tables, with Pass/Fail QC checks.
+
+### :snake: Rule
+
+#SNAKEMAKE_RULE_SOURCE__somalier__create_somalier_mqc_tsv#
```

markdownlint-cli2 (0.22.1)

[warning] No space inside hashes on closed atx style heading

(MD020, no-missing-space-closed-atx)